src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,122 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    direction_placeholder: T('Direction'),
+    direction_tooltip: T(''),
+
+    transport_placeholder: T('Transport'),
+    transport_tooltip: T(''),
+
+    ssh_credentials_placeholder: T('SSH Connection'),
+    ssh_credentials_tooltip: T(''),
+
+    netcat_active_side_placeholder: T('Netcat Active Side'),
+    netcat_active_side_tooltip: T(''),
+
+    netcat_active_side_listen_address_placeholder: T('Netcat Active Side Listen Address'),
+    netcat_active_side_listen_address_tooltip: T(''),
+
+    netcat_active_side_port_min_placeholder: T('Netcat Active Side Min Port'),
+    netcat_active_side_port_min_tooltip: T(''),
+
+    netcat_active_side_port_max_placeholder: T('Netcat Active Side Max Port'),
+    netcat_active_side_port_max_tooltip: T(''),
+
+    netcat_passive_side_connect_address_placeholder: T('Netcat Active Side Connect Address'),
+    netcat_passive_side_connect_address_tooltip: T(''),
+
+    source_datasets_placeholder: T('Source Datasets'),
+    source_datasets_tooltip: T(''),
```

Comment:
This can now accept multiple source datasets
```
    # Or you can specify multiple source datasets, e.g.:
    # source-dataset:
    #   - data/src/work
    #   - data/src/holiday/summer
    # They would be replicated to data/dst/work and data/dst/holiday/summer
```

Reply:
Updated help text:

From the source computer with snapshots to replicate, choose an existing ZFS pool or dataset with an active periodic snapshot task. Multiple source datasets can be chosen.